token-account-based/chaincode-go/chaincode/token_contract.go

```diff
@@ -101,6 +102,14 @@ func (s *SmartContract) Transfer(ctx contractapi.TransactionContextInterface, re
 		return fmt.Errorf("failed to transfer: %v", err)
 	}
 
+	// Emit the Transfer event
+	var transferEvent = map[string]string{"from": clientID, "to": recipient, "value": strconv.Itoa(amount)}
```

Comment:
nit: Objects with known values are generally defined using `:=` unless the object is in the global scope
```
transferEvent := map[string]string{"from": clientID, "to": recipient, "value": strconv.Itoa(amount)}
```

Reply:
👌🏼